eva/udfs/new_udf_impl/base_udf_fns.py

```diff
@@ -0,0 +1,59 @@
+
+import numpy as np
+import pandas as pd
+import torch
+from facenet_pytorch import MTCNN
```

Comment:
the fundamental problem with this implementation
is that it does not work for user code
the point of a decorator is that the user can decorate their own functions
and then run a SQL command
to ingest the UDF
if you have an if statement that checks the model type in the base_udf_fns.py file
this is not possible